utils/utils.go

```diff
@@ -0,0 +1,63 @@
+package utils
```

Comment:
I don't agree with having a package named "utils". This is for sure not best practice. What's the responsibility of this package? Why are those functions in this package and not others? Why should these functions be public in the whole app? We only need them when parsing the yml-files. They belong in the actions package as private members in my opinion.

Reply:
You are right, the naming convention is not the best.
This package includes all utilities we are going to need around other packages.
At the end I will split this package into small more focused ones.